.github/ISSUE_TEMPLATE/syncer-support.md

```diff
@@ -48,3 +48,4 @@ Additionally, please provide the following info before submitting your issue. Th
 3. Common issues
     - [ ] Is the `worker-count` set to a reasonable value (suggest-32)?
     - [ ] Is the `batch` set to a reasonable value (suggest 100-1000)?
+    - [ ] Is the downstream server mysql and using multiStatements or not utf8mb4 charset (syncer can only provide limited support for MySQL as a downstream server)?
```

Comment:
This is a bit hard to parse (`(a and b) or c`? `a and (b or c)`?). And I suppose `multiStatements` should be turned on, and the character set should be set to `utf8mb4`? Perhaps split into two items if there are two thing to check.